cob_bringup/robots/base_solo.launch

```diff
@@ -16,11 +17,12 @@
 	<!-- start robot_state_publisher -->
 	<node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher"/>
 
-	<include unless="$(arg use_old_base_drive_chain)" file="$(find cob_bringup)/components/cob4_base.launch">
+	<include unless="$(arg use_old_base_drive_chain)" file="$(find cob_bringup)/bundles/base_cob4.launch"/>
+		<arg name="base" value="$(arg base)"/>
 	</include>
 
-	<include if="$(arg use_old_base_drive_chain)" file="$(find cob_bringup)/components/legacy_base.launch">
-		<arg name="robot" value="$(arg robot)"/>
+	<include if="$(arg use_old_base_drive_chain)" file="$(find cob_bringup)/bundles/base_legacy.launch">
```

Comment:
currently it's us not moving on here....the only robots using legacy base are `raw3-1` and `raw3-3` - `raw3-5` could be used as a template..see #417